app/src/main/java/com/space/quizapp/presentation/ui/start/custom_view/StartVectorView.kt

```diff
@@ -0,0 +1,75 @@
+package com.space.quizapp.presentation.ui.start.custom_view
+
+import android.content.Context
+import android.graphics.Canvas
+import android.graphics.Paint
+import android.graphics.Path
+import android.graphics.RectF
+import android.util.AttributeSet
+import android.view.View
+import androidx.core.content.ContextCompat
+import com.space.quizapp.R
+
+/**
+ * A custom view that draws a blue vector on the top and a white vector on the bottom.
+ */
+class StartVectorView @JvmOverloads constructor(
+    context: Context,
+    attrs: AttributeSet? = null,
+    defStyleAttr: Int = 0
+) : View(context, attrs, defStyleAttr) {
+
+    private val fillPaint = Paint().apply {
+        isAntiAlias = true
+        style = Paint.Style.FILL
+    }
+
+    private val path1 by lazy { Path() }
+    private val path2 by lazy { Path() }
```

Comment:
path1, path2 -ს ჯობია  path რას ხატავს, იმის სახელები დაარქვა

Reply:
done in this pr: https://github.com/iTarley/QuizApp-SpaceIntern/pull/6